services/venice-router/src/test/java/com/linkedin/venice/router/api/TestDictionaryRetrievalService.java

```diff
@@ -3,15 +3,14 @@
 import static com.linkedin.venice.router.api.DictionaryRetrievalService.MAX_DICTIONARY_DOWNLOAD_DELAY_TIME_MS;
 import static com.linkedin.venice.router.api.DictionaryRetrievalService.MIN_DICTIONARY_DOWNLOAD_DELAY_TIME_MS;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.anyInt;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.*;
```

Comment:
could you please replace this with specific imports?

Reply:
Yup, reverted.